flink-streaming-java/src/main/java/org/apache/flink/streaming/api/datastream/KeyedStream.java

```diff
@@ -173,67 +174,67 @@ public KeyedStream(DataStream<T> dataStream, KeySelector<T, KEY> keySelector, Ty
 	}
 
 	/**
-	 * Applies the given {@link TimelyFlatMapFunction} on the input stream, thereby
+	 * Applies the given {@link ProcessFunction} on the input stream, thereby
 	 * creating a transformed output stream.
 	 *
 	 * <p>The function will be called for every element in the stream and can produce
 	 * zero or more output. The function can also query the time and set timers. When
 	 * reacting to the firing of set timers the function can emit yet more elements.
 	 *
```

Comment:
<p>The function will be called for every element in the input streams and can produce zero or more
	 * output elements. Contrary to the {@link #flatMap(FlatMapFunction)} function, this function can also
	 * query the time and set timers. When reacting to the firing of set timers the function can directly
	 * emit elements and/or register yet more timers.